src/run.rs

```diff
@@ -303,6 +304,63 @@ fn image_ids_in_use() -> io::Result<HashSet<String>> {
     Ok(image_ids)
 }
 
+// Find docker root directory.
+fn docker_root_dir() -> io::Result<String> {
+    // Query Docker for the root directory.
+    let output = Command::new("docker")
+        .args(&["info", "--format", "{{.DockerRootDir}}"])
+        .stderr(Stdio::inherit())
+        .output()?;
+
+    // Ensure the command succeeded.
+    if !output.status.success() {
+        return Err(io::Error::new(
+            io::ErrorKind::Other,
+            "Unable to determine the Docker root directory.",
+        ));
+    }
+
+    // Find the relevant line of output.
+    String::from_utf8(output.stdout)
+        .map_err(|error| io::Error::new(io::ErrorKind::Other, error))
+        .map(|s| s.trim().to_string())
+}
+
+// Get number of matching characters at the start of two strings.
+fn prefix_match_len(s1: &str, s2: &str) -> usize {
+    let size = std::cmp::min(s1.len(), s2.len());
```

Comment:
Since we're already importing `std::cmp::max` at the top of the file, maybe we can do the same for `std::cmp::min` and just refer to it as `min` here.